modular_skyrat/modules/goofsec/code/projectiles.dm

```diff
@@ -123,9 +123,8 @@
 	fire_sound = 'sound/weapons/gun/smg/shot.ogg'
 
 /obj/projectile/beam/laser/hardlight_bullet/ostwind
-	damage = 27 // Minimum damage increase to at least base /tg/ laser gun damage.
-	embedding = list(embed_chance=10, fall_chance=3, jostle_chance=4, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=5, jostle_pain_mult=6, rip_time=10)
-
+	damage = 30 // 11/20/21: Increased from 27 to 30 to improve underperforming guns.
```

Comment:
this makes the ostwind down in about 5s, equal to the bulldog with buckshot from 2 tiles